vmware_aria_operations_integration_sdk/mp_init.py

```diff
@@ -355,7 +377,7 @@ def main() -> None:
                 vendor,
                 eula_file,
                 icon_file,
-                language,
+                adapter_config,
                 template_style,
```

Comment:
Since the available template are going to be (potentially) language-dependent, we should consider moving that (and the prompt for them) into the AdapterConfig class in a future PR.